demos/SSHRemoting/README.md

```diff
@@ -152,10 +168,11 @@ GitCommitId                    v6.0.0-alpha.17
 ```
 
 ### Known Issues:
-1.  You can currently establish a connection either interactively with user name and password or via key authentication.  PSCredential is not yet supported.
-2.  Remote debugging from Linux to Windows does not work.  However, remote debugging from Windows to Linux does work.
-3.  Fan out to multiple machines not yet supported.
-4.  sudo command does not work in remote session to Linux machine.
+1. You can currently establish a connection either interactively with user name and password or via key authentication.  
+2. PSCredential is not yet supported.
```

Comment:
Actually it is unlikely PSCredential will be supported in the foreseeable future because it needs to be adopted by OpenSSH and there is currently much disagreement about whether it should be supported.  I feel we should just remove item 2. and not imply it is forthcoming.